### PR TITLE
Update device-enrollment-program-enroll-ios.md

### PR DESCRIPTION
**Pull request by msbemba**
Added more information to reduce ambiguity

Fixes https://github.com/MicrosoftDocs/memdocs/issues/4270

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: memdocs/intune/enrollment/device-enrollment-program-enroll-ios.md [hunk 1]
```diff
@@ -464,7 +464,7 @@ You'll sometimes need to renew your tokens:
 - Renew your ADE token yearly. The Intune admin center shows the expiration date.
 - If the Apple ID password changes for the user who set up the token in Apple Business Manager, renew your enrollment program token in Intune and Apple Business Manager.
 - If the user who set up the token in Apple Business Manager leaves the organization, renew your enrollment program token in Intune and Apple Business Manager.
-- If you change the Apple ID used to create the ADE token, the change will not affect currently enrolled devices.
+- If you change the Apple ID used to create the ADE token, the change will not affect currently enrolled devices. This is Unlike APNS Certificate which cannot be changed without requiring all devices to be re-enrolled unless you contact Apple Support to perform the change on the back end.
```
Comment (JohanFreelancer9) [inline memdocs/intune/enrollment/device-enrollment-program-enroll-ios.md:467]:
```suggestion
- If you change the Apple ID used to create the ADE token, the change will not affect currently enrolled devices. This is unlike an Apple Push Notification Service (APNS) certificate which cannot be changed without requiring all devices to be re-enrolled unless you contact Apple Support to perform the change on the back end. 
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review..

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@lenewsad Please review and sign off this PR. Thanks

**Comment by lenewsad** [comment]
Hi @msbemba, this is pending PM approval.

**Review by lenewsad — APPROVED** [review]

**Comment by lenewsad** [comment]
Thank you @msbemba! 

#sign-off

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@tiburd

**Pull request merged**